azure/src/test/java/com/microsoft/azure/AzureTests.java

```diff
@@ -76,37 +72,6 @@ public void setup() throws Exception {
     }
 
     /**
-     * Tests the public IP address implementation from an individual service client
-     * @throws Exception 
-     */
-    @Test
-    public void testPublicIpAddressesInGroup() throws Exception {
```

Comment:
Deleting it would make it hard to track.
Replace with this?
```java
    @Test
    public void testPublicIpAddressesInGroup() throws Exception {
        final String suffix = String.valueOf(System.currentTimeMillis());
        final String newGroupName = "group" + suffix;
        final String newPipName = "pip" + suffix;
        ResourceGroup group = azure2.resourceGroups().define(newGroupName)
                .withRegion(Region.US_WEST)
                .create();

        PublicIpAddresses pips = azure2.publicIpAddresses();
        
        PublicIpAddress pip = pips.define(newPipName)
            .withRegion(Region.US_WEST)
            .withExistingGroup(group)
            .withDynamicIp()
            .withLeafDomainLabel(newPipName)
            .create();
        
        System.out.println("Public IP addresses count: " + pips.list(group.name()).size());
        
        pip.update()
            .withLeafDomainLabel(newPipName + "x")
            .apply();
        
        pips.delete(group.name(), pip.name());
        azure2.resourceGroups().delete(group.name());
    }
```